### PR TITLE
Alter fdw rake task to correctly set up foreign data wrapper

### PR DESCRIPTION
**Pull request by drubio1989**

### CODE REVIEW

File: lib/tasks/fix_fdw_credentials.rake [hunk 1]
```diff
@@ -6,31 +22,33 @@ end
 
 def fix_primary_server_fdw_credentials
   config = ActiveRecord::Base.configurations['secondbase'][Rails.env]
+  username = ActiveRecord::Base.configurations[Rails.env]['username']
```
Comment (msk999) [inline lib/tasks/fix_fdw_credentials.rake]: Have the .env files with this new configuration already been pushed to master?

Comment (drubio1989) [reply lib/tasks/fix_fdw_credentials.rake]: Yeah, they are in the openhub-config directory and are updated.

**Comment by drubio1989** [comment]
retest this please.

**Comment by drubio1989** [comment]
retest this please.

**Pull request merged**